safe_transaction_service/history/services/notification_service.py

```diff
@@ -204,3 +205,55 @@ def build_reorg_payload(block_number: int) -> ReorgPayload:
         blockNumber=block_number,
         chainId=str(get_chain_id()),
     )
+
+
+class DelegatePayload(TypedDict):
+    type: str
+    safeAddress: Optional[str]
+    delegate: str
+    delegator: str
+    label: str
+    expiryDateSeconds: Optional[int]
```

Comment:
All of the existing events have a chain identifier. I suggest we add a `chainId` too, for example.

Reply:
Yes, you are right. Good catch! 

Done https://github.com/safe-global/safe-transaction-service/pull/2310/commits/bd6310ee83c17efb76b4b1e0fbfd84775e71cdef